app/src/main/res/values/strings.xml

```diff
@@ -620,6 +620,16 @@
     <!-- Content description (not visible, for screen readers etc.): "Close button for library settings" -->
     <string name="content_description_close_button">Close</string>
 
+    <!-- Title to show in alert when a lot of tabs are to be opened -->
+    <string name="open_all_warning_title">Confirm open</string>
+    <!-- Message to warn users that a large number of tabs will be opened
+    %d is a placeholder for the number of tabs that will be opened. -->
+    <string name="open_all_warning_message">You are about to open %d tabs. This might slow down Firefox while the pages are loading. Are you sure you want to continue?</string>
```

Comment:
This strings are the same that's used in desktop.  This will need to be reviewed.